pkg/operator/workload/sync_openshift_oauth_apiserver.go

```diff
@@ -102,6 +107,12 @@ func (c *OAuthAPIServerWorkload) PreconditionFulfilled() (bool, error) {
 		klog.Info("Waiting for observed configuration to be available")
 		return false, errors.New("waiting for observed configuration to be available (haven't found APIServerArguments in spec.ObservedConfig.Raw)")
 	}
+
+	// specifying etcd servers list is mandatory, without it the pods will be crashlooping, so wait for it.
+	if storageServers, _ := operatorCfg.APIServerArguments[libgoetcd.StorageConfigURLsKey]; len(storageServers) == 0 {
+		klog.Infof("Waiting for observed configuration to have mandatory %q", libgoetcd.StorageConfigURLsKey)
+		return false, errors.New(fmt.Sprintf("waiting for observed configuration to have mandatory %q", libgoetcd.StorageConfigURLsKey))
```

Comment:
"waiting for observed configuration field `apiServerArguments.%s` to be set"

maybe?

Reply:
sure, np

Reply:
better be explicit